src/main/avro/GuessService.avdl

```diff
@@ -0,0 +1,27 @@
+@version("1.0")
+@namespace("com.github.rakhmedovrs.avro")
+protocol GuessService {
+
+	enum RequestIntention {
+		MAKE_A_GUESS,
+		DROP_CONNECTION
+	}
+
+	enum GuessResult {
+		CORRECT,
+		INCORRECT,
+		UNDEFINED
+	}
+
+	record ClientGuessRequest {
```

Comment:
Since it can be DROP_CONNECTIO, perhaps it is not a GuessRequest anymore, but more like a Request or ClientRequest.

Reply:
Fixed